src/components/AnalysisModal.js

```diff
@@ -105,38 +118,32 @@ export const AnalysisModal = Utils.withDisplayName('AnalysisModal')(
       persistentDisks,
       location,
       onDismiss: () => {
-        resetView()
+        setViewMode(undefined)
```

Comment:
I made this change to be consistent with https://github.com/DataBiosphere/terra-ui/blob/c0ceb2e8ff4e7700e43f981126d3feb58ce61f97/src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js#L63, but I don't know if resetView is more appropriate here (`setViewMode(undefined)`) is what we are also calling below in the similar code for GalaxyModal.

Reply:
As we discussed, I changed to call resetView here and the other similar locations in this file.